modules/phase_field/test/tests/Grain_Velocity_Computation/GrainBoundaryVelocityTest.i

```diff
@@ -0,0 +1,137 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+  nz = 0
+  xmin = 0
+  xmax = 1000
+  ymin = 0
+  ymax = 1000
+  zmin = 0
+  zmax = 0
+  elem_type = QUAD4
+
+  uniform_refine = 2
+[]
+
+[GlobalParams]
+  op_num = 2
+  var_name_base = gr
+  v = 'gr0 gr1'
+[]
+
+[Variables]
+  [./gr0]
+    order = FIRST
+    family = LAGRANGE
```

Comment:
you can leave out family and order, they default to the values you're setting. Shorter files are more readable.